File: Training Exercises/Back From The Park/sample.java
import java.util.Scanner;

public class Main {
	static Scanner sc = new Scanner(System.in);

	public static void main(String[] args) {
		int x = sc.nextInt();
		sc.nextInt();
		int xx = sc.nextInt();
		sc.nextInt();
		if (xx > x)
			System.out.println("Right");
		else
			System.out.println("Left");
	}

}

File: Training Exercises/Back From The Park/sample.go
package main

import "fmt"

func main() {
	var x , y , x1 , y1 int
	fmt.Scan(&x , &y)
	fmt.Scan(&x1 , &y1)
	if x1 < x {
		fmt.Println("Left")
	} else{
		fmt.Println("Right")
	}
}
